integration_testing/features/cancel-copy-of-multiple-selections.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Cancel the copy of multiple selections
+
+	Background:
+		Given I am signed in to Studio
+			And I am on the channel editor view
+		Then I click on clipboard floating action button on the bottom-right of the screen

```

Comment:
Not sure this is exactly kosher, let's try to reformulate this so *Then* belongs to the *Scenario* instead of the *Given*.